content/rs/references/compatibility/breaking-changes.md

```diff
@@ -0,0 +1,55 @@
+---
+Title: Breaking changes
```

Comment:
maybe we should emphasize this is 'redis breaking changes' (as opposed to Redis Enterprise other features)

Reply:
Updated the title